src/containers/edit-profile/notification-tab/NotificationTab.tsx

```diff
@@ -7,30 +7,46 @@ import SettingItem from '~/components/setting-item/SettingItem'
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import { titleWithSubtitle } from '~/containers/edit-profile/notification-tab/NotificationTab.constants'
 import { styles } from '~/containers/edit-profile/notification-tab/NotificationTab.styles'
+import AppButton from '~/components/app-button/AppButton'
+import { ButtonVariantEnum, SizeEnum } from '~/types'
 
 const NotificationTab = () => {
   const { t } = useTranslation()
 
+  const handleUpdateData = () => {
+    // @TODO: implement update data logic
+  }
+
   return (
-    <Box sx={styles.root}>
-      <TitleWithDescription
-        description={t('editProfilePage.profile.notificationsTab.mainHint')}
-        style={styles.titleWithDescription}
-        title={t('editProfilePage.profile.notificationsTab.notifications')}
-      />
+    <Box sx={styles.notificationInnerContainer}>
+      <Box sx={styles.root}>
+        <TitleWithDescription
+          description={t('editProfilePage.profile.notificationsTab.mainHint')}
+          style={styles.titleWithDescription}
+          title={t('editProfilePage.profile.notificationsTab.notifications')}
+        />
 
-      <Box sx={styles.optionsContainer}>
-        {titleWithSubtitle.map((item) => (
-          <SettingItem
-            key={item.title}
-            style={styles.options}
-            subtitle={t(item.subtitle)}
-            title={t(item.title)}
-          >
-            <Switch sx={styles.switch} />
-          </SettingItem>
-        ))}
+        <Box sx={styles.optionsContainer}>
+          {titleWithSubtitle.map((item) => (
+            <SettingItem
+              key={item.title}
+              style={styles.options}
+              subtitle={t(item.subtitle)}
+              title={t(item.title)}
+            >
+              <Switch sx={styles.switch} />
+            </SettingItem>
+          ))}
```

Comment:
I suggest moving this map before `return` statement

Reply:
Good idea, fixed